lib/routes/devices-and-sessions.js

```diff
@@ -161,12 +173,14 @@ module.exports = (log, db, config, customs, push, devices) => {
           payload: isA.alternatives().try(
             isA.object({
               to: isA.string().valid('all').required(),
+              endpointAction: isA.string().valid('accountVerify').optional(),
```

Comment:
Do we have an established pattern for testing-related request parameters like this?  I'm thinking something like "_endpointAction" to make it more obvious to calling code that it probably shouldn't be specifying this unless it really knows what it's doing.

Not a big deal though.